pxf/HivePXF.html.md.erb

```diff
@@ -151,184 +477,120 @@ To enable HCatalog query integration in HAWQ, perform the following steps:
     postgres=# GRANT ALL ON PROTOCOL pxf TO "role";
     ``` 
     
-3.  To query a Hive table with HCatalog integration, simply query HCatalog directly from HAWQ. The query syntax is:
 
-    ``` sql
-    postgres=# SELECT * FROM hcatalog.hive-db-name.hive-table-name;
-    ```
+To query a Hive table with HCatalog integration, query HCatalog directly from HAWQ. The query syntax is:
```

Comment:
It's a bit awkward to drop out of the procedure and into free-form discussion of the various operations.  I think it might be better to put the previous 3-step procedure into a new subsection like "Enabling HCatalog Integration" and then putting the remaining non-procedural content into "Usage" ?